### ISSUE TITLE
Align API version and package version

### ISSUE THREAD

**Issue by dvrogozh** [intel/intel-graphics-compiler#80]
I've built latest code and got library named "libiga64.so.1.0.0", but igc-opencl.pc has this line:
`Version: 0.1.0`

So, we have 2 versions now. Here is API version:
https://github.com/intel/intel-graphics-compiler/blob/17d9e620901e1e2c863190f2996867e07ea0602a/IGC/CMakeLists.txt#L57

and here is another version:
https://github.com/intel/intel-graphics-compiler/blob/17d9e620901e1e2c863190f2996867e07ea0602a/IGC/CMakeLists.txt#L1874

The latter one is used in igc-opencl.pc.in:
https://github.com/intel/intel-graphics-compiler/blob/17d9e620901e1e2c863190f2996867e07ea0602a/IGC/AdaptorOCL/igc-opencl.pc.in#L8

I suggest to either completely remove "another" version or at least default it to API version like:
```
if(NOT DEFINED MAJOR_VERSION)
  set(MAJOR_VERSION ${IGC_API_MAJOR_VERSION})
endif()
if(NOT DEFINED MINOR_VERSION)
  set(MINOR_VERSION ${IGC_API_MINOR_VERSION})
endif()
if(NOT DEFINED PATCH_VERSION)
  set(PATCH_VERSION ${IGC_API_PATCH_VERSION})
endif()
```

**Comment by paigeale** [comment]
Thanks Dmitry. Good catch will fix this.

**Comment by paigeale** [comment]
Issue addressed with 49497e5d5acc01a6aaa77907599eb16f335e8e6e.

**Comment by dvrogozh** [comment]
@paigeale : I don't see that you have changed https://github.com/intel/intel-graphics-compiler/blob/17d9e620901e1e2c863190f2996867e07ea0602a/IGC/AdaptorOCL/igc-opencl.pc.in#L8.
